src/classes/RD2_Constants.cls

```diff
@@ -53,6 +53,15 @@ public inherited sharing class RD2_Constants {
         Disable_All_Installments
     }
 
+    /**
+    * @description Enum to specify asynchronous behavior for the creation of the first installment opportunity
+    */
+    public enum FirstInstallmentAsyncOptions {
+        Synchronous_Always,
```

Comment:
To match the help text on the field, this should be:

- Synchronous
- Asynchronous
- Asynchronous_When_Bulk